libraries/AP_HAL_ChibiOS/hwdef/common/stm32h7_mcuconf.h

```diff
@@ -607,12 +607,16 @@
 /*
  * USB driver system settings.
  */
+#ifndef STM32_USB_USE_OTG1
 #define STM32_USB_USE_OTG1                  TRUE
+#endif
+#ifndef STM32_USB_USE_OTG2
 #define STM32_USB_USE_OTG2                  TRUE
+#endif
 #define STM32_USB_OTG1_IRQ_PRIORITY         14
 #define STM32_USB_OTG2_IRQ_PRIORITY         14
-#define STM32_USB_OTG1_RX_FIFO_SIZE         512
-#define STM32_USB_OTG2_RX_FIFO_SIZE         1024
+#define STM32_USB_OTG1_RX_FIFO_SIZE         2048
```

Comment:
This is a significant change, think it should be dependent on using OTG_HS

Reply:
for stm32h7 chips this is not a huge cost, I don't think its needed to have ifdefs around here.